src/App.js

```diff
@@ -15,6 +15,41 @@ import * as actions from './store/actions';
 import  { Redirect } from 'react-router-dom';
 
 
+import firebase from 'firebase';
+var config = {
+    apiKey: "AIzaSyBibrWw6b833N4ar2-wY5wIT3TT1IR_pyg",
+    authDomain: "saharok-221817.firebaseapp.com",
+    databaseURL: "https://saharok-221817.firebaseio.com",
+    projectId: "saharok-221817",
+    storageBucket: "saharok-221817.appspot.com",
+    messagingSenderId: "406120965570"
+};
```

Comment:
Предлагаю придерживаться единого стиля относительно работы с объявлением переменных (в данном случае: `let`, `const`, но не `var`) :)

```suggestion
const config = {
    apiKey: "AIzaSyBibrWw6b833N4ar2-wY5wIT3TT1IR_pyg",
    authDomain: "saharok-221817.firebaseapp.com",
    databaseURL: "https://saharok-221817.firebaseio.com",
    projectId: "saharok-221817",
    storageBucket: "saharok-221817.appspot.com",
    messagingSenderId: "406120965570"
};
```